website/docs/docs/quickstarts/dbt-core/codespace-qs.md

```diff
@@ -0,0 +1,72 @@
+1. Wait for the codespace to finish building by waiting for the `postCreateCommand` command to complete; this can take several minutes:
+
+    <Lightbox src="/img/codespace-quickstart/postCreateCommand.png" title="Wait for postCreateCommand to complete" />
+
+    When this command completes, you can start using the codespace development environment. The terminal the command ran in will close and you will get a prompt in a brand new terminal. 
+
+1. At the terminal's prompt, you can execute any dbt command you want. For example:
+
+    ```shell
+    /workspaces/test (main) $ dbt build
+    ```
+
+    You can also use the [duckcli](https://github.com/dbcli/duckcli) to write SQL against the warehouse from the command line or build reports in the [Evidence](https://evidence.dev/) project provided in the `reports` directory.
+    
+    For complete information, refer to the [dbt command reference](https://docs.getdbt.com/reference/dbt-commands). Common commands are:
+    
+    - [dbt compile](https://docs.getdbt.com/reference/commands/compile) — generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) — compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) — compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) — compiles, runs, and tests your project
+
+## Generate a larger data set
+
+If you'd like to work with a larger selection of Jaffle Shop data, you can generate an arbitrary number of years of fictitious data from within your codespace. 
+
+1. Install the Python package called [jafgen](https://pypi.org/project/jafgen/). At the terminal's prompt, run:
+
+    ```shell
+    /workspaces/test (main) $ pip install jafgen
+    ```
+
+1. When installation is done, run:
+    ```shell
+    /workspaces/test (main) $ jafgen --years NUMBER_OF_YEARS
+    ``` 
+    Replace `NUMBER_OF_YEARS` with the number of years you want to simulate. This command builds the CSV files and stores them in the `jaffle-data` folder, and is automatically sourced based on the `sources.yml` file and the `dbt-duckdb` adapter.
+
+As you increase the number of years, it takes exponentially more time to generate the data because the Jaffle Shop stores grow in size and number. For a good balance of data size and time to build, dbt Labs suggests a maximum of 6 years.
+
+## Related content
+- [Create a GitHub repository](/docs/quickstarts/dbt-core/manual-install#create-a-repository)
+- [Build your first models](/docs/quickstarts/dbt-core/manual-install#build-your-first-models)
+- [Test and document your project](/docs/quickstarts/dbt-core/manual-install#test-and-document-your-project)
+- [Schedule a job](/docs/quickstarts/dbt-core/manual-install#schedule-a-job)
```

Comment:
are the 4 links above appropriate? is the list missing anything (like `/docs/quickstarts/dbt-core/manual-install#create-a-project`)?